db/config/index.js

```diff
@@ -14,7 +16,7 @@ module.exports = {
     logging: false
   }),
 
-  ...(process.env.NEXT_DB_HOST
+  ...(dbHost && dbHost !== "localhost"
```

Comment:
must add a variable to _turn off_ ssl -- as with localhost usually it's not needed